src/ontology/uberon-edit.obo

```diff
@@ -8749,6 +8749,7 @@ xref: VHOG:0000098
 xref: Wikipedia:Endocrine_system
 xref: XAO:0000158
 xref: ZFA:0001158
+is_a: UBERON:0015203 ! non-connected functional system
```

Comment:
In #3043, the 'neuroendocrine system' is classified as an 'anatomical system', which has a text def asserting it to be a "connected anatomical structure".

The definition of the PATO class [disconnected](http://purl.obolibrary.org/obo/PATO_0010001) is "A structural quality inhering in the bearer by virtue of the bearer consisting of multiple structures lacking any physical connection to each other."

I'm not sure if the above means at a single instant in time. If not, one may argue that physically distant structures are connected over time via the release and binding of hormones, neurotransmitters, etc.

Either way, 'endocrine system' and neuroendocrine system' should be modelled similarly.

Reply:
I agree that however “endocrine system” is classified, “neuroendocrine system” should be classified similarly.

FWIW, I think interpreting “connected” as “connected over space OR time” is a bit of a stretch, especially since there is nothing in the definition of all the related terms that could suggest such a meaning. For example, [anatomical structure](http://purl.obolibrary.org/obo/UBERON_0000061) (which in Uberon is the root class for all things connected – including [anatomical system](http://purl.obolibrary.org/obo/UBERON_0000467)) is defined as “material anatomical entity that is a single connected structure with inherent 3D shape” – I don’t see how this leaves room for involving time.

In any case, the connected vs disconnected issue has been a regular source of unsatisfiability (especially when merging Uberon with other ontologies, because some other ontologies not only have a distinction between connected and disconnected but also _enforce_ that distinction through disjointness axioms, something that Uberon does not), so any change to the meaning of “(dis)connected” should be discussed carefully in a dedicated ticket.

This PR merely restores “endocrine system” as a “non-connected functional system”, as it was until last week.